### PR TITLE
[11.0][MIG] sale_force_invoiced

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

### CODE REVIEW

File: sale_force_invoiced/model/sale_order.py [hunk 1]
```diff
@@ -8,20 +7,19 @@
...
 
     @api.depends('force_invoiced')
     def _get_invoiced(self):
         super(SaleOrder, self)._get_invoiced()
-        for order in self:
-            if order.force_invoiced and order.invoice_status == 'to invoice':
-                order.invoice_status = 'invoiced'
+        self.filtered(lambda so: so.force_invoiced
+                      and so.invoice_status == 'to invoice').write(
+            {'invoice_status': 'invoiced'})
```
Comment (LoisRForgeFlow) [inline sale_force_invoiced/model/sale_order.py]: Revert this change, you cannot use write inside an api.depends.

**Review by LoisRForgeFlow — CHANGES_REQUESTED** [review]

File: sale_force_invoiced/model/sale_order.py (hunk 1 above)
Comment (MiquelRForgeFlow) [reply sale_force_invoiced/model/sale_order.py]: done

**Review by hieulucky111 — APPROVED** [review]

**Review by LoisRForgeFlow — APPROVED** [review]
:+1:

File: sale_force_invoiced/model/sale_order.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+class SaleOrder(models.Model):
+    _inherit = 'sale.order'
+
+    force_invoiced = fields.Boolean(
+        string='Force invoiced',
+        help='When you set this field, the sales order will be considered as '
+             'fully invoiced, even when there may be ordered or delivered '
+             'quantities pending to invoice.',
+        readonly=True,
+        states={'done': [('readonly', False)]},
```
Comment (espo-tony) [inline sale_force_invoiced/model/sale_order.py:16]: I'm wondering if it shouldn't be editable on _sale_ state rather than on _done_ state (or maybe on both).

**Review by espo-tony — COMMENTED** [review]
Code review LGTM. I just have a question about how this module should be used.

File: sale_force_invoiced/model/sale_order.py (hunk 2 above)
Comment (rafaelbn) [reply sale_force_invoiced/model/sale_order.py:16]: Please @mreficent could you answer this doubt? thanks
